transposon/analyzeBCcounts.sh

```diff
@@ -172,7 +173,7 @@ if [ "${minCellBCLength}" -gt 0 ]; then
         #June 2020 scaled
         scRNAseqbase="/vol/mauranolab/transposon/scrnaseq/merged_T0219toT0222"
         #/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_conflicting_cells.txt
-        genotypeClonesArgs="--blacklist /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_multisite.txt,/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_shared.txt --annotateclones /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/bc_transfection-occurrence.txt.gz --splitConflictingClones transfection"
+        genotypeClonesArgs="--blacklist /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_multisite.txt,/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_shared.txt --annotateclones /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/bc_transfection-occurrence.txt.gz"
```

Comment:
accidental reversion?

Reply:
No, since we move out of --splitConflictingClones [key]. I didn't want to keep it in the new proposal

Reply:
Not sure I follow -- this PR should result in your new filters being applied to T0219toT0221

Reply:
Andre, you should specify the parameters for this transfection here:
--transfectionKey transfection --removeMinorityBCsFromConflictingCells 0.xx --removeConflictingCells

(I forget what rate you chose for removeMinorityBCsFromConflictingCells)